Jenkinsfile

```diff
@@ -197,17 +178,11 @@ node('vetsgov-general-purpose') {
     try {
       def builds = [:]
 
-      for (int i=0; i<envNames.size(); i++) {
-        def envName = envNames.get(i)
-
-        def cmsDeployArg = ""
-        if (isCMSDeploy) {
-          cmsDeployArg = "--content-deployment"
-        }
-
+      for (int i=0; i<VAGOV_BUILDTYPES.size(); i++) {
+        def envName = VAGOV_BUILDTYPES.get(i)
         builds[envName] = {
           dockerImage.inside(args) {
-            sh "cd /application && npm --no-color run build -- ${cmsDeployArg} --buildtype=${envName}"
+            sh "cd /application && npm --no-color run build -- --buildtype=${envName}"
             sh "cd /application && echo \"${buildDetails('buildtype': envName, 'ref': ref)}\" > build/${envName}/BUILD.txt"
           }
         }
```

Comment:
I also removed the CMS argument altogether, since we need a direct-to-prod build-type for it to mean anything. That work will be tracked here, https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14855.